mitmproxy/flow/export.py

```diff
@@ -4,23 +4,34 @@
 import re
 from textwrap import dedent
 
-from six.moves.urllib.parse import quote, quote_plus
+import six
+from six.moves import urllib
 
 import netlib.http
 
 
+def _native(s):
```

Comment:
This function is exactly the same as [`strutils.native`](https://github.com/mitmproxy/mitmproxy/blob/master/netlib/strutils.py#L14-L29) except for [this part](https://github.com/mitmproxy/mitmproxy/blob/master/netlib/strutils.py#L21-L22), as we need this here because s *could* be lists etc.